README.md

```diff
@@ -1,39 +1,40 @@
-[![GitHub issues](https://img.shields.io/github/issues/ludens-reklamebyra/react-womb.svg)](https://github.com/ludens-reklamebyra/react-womb/issues) ![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)
+[![GitHub issues](https://img.shields.io/github/issues/ludens-reklamebyra/react-crisscross.svg)](https://github.com/ludens-reklamebyra/react-crisscross/issues) ![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)
 
-<h1 align="center">React Womb :dizzy:</h1>
+<h1 align="center">React crisscross :triangular_ruler:</h1>
 <p align="center">A simple and flexible layout component for React based on CSS flexbox</p>
 
 ## Layout component
 
 ```jsx
 <Layout container>
-  <Layout item>
-    {/* Content */}
-  </Layout>
+  <Layout item>{/* Content */}</Layout>
```

Comment:
Should be as is?

Reply:
Just to make that absolutely clear.. It's my prettier config.